alexandria/core/tests/test_migration.py

```diff
@@ -0,0 +1,65 @@
+import uuid
+
+import pytest
+
+
+def is_valid_uuid(val):
+    try:
+        uuid.UUID(str(val))
+        return True
+    except ValueError:  # pragma: no cover
+        return False
+
+
+@pytest.mark.django_db()
+def test_tag_pk_migration(migrator):
```

Comment:
Had to dig a bit, but: In other projects, we built a `post_migrate_to_current_state` fixture that resets the test DB to the newest version after such a test. The `django-test-migrations` package seems to do this for us, so nice find! (CC @open-dynaMIX @fugal-dy, we should totally replace our ugly fixture with this!)